packages/hint-no-broken-links/src/hint.ts

```diff
@@ -32,13 +32,21 @@ export default class NoBrokenLinksHint implements IHint {
             description: `Hint to flag broken links in the page`
         },
         id: 'no-broken-links',
-        schema: [],
+        schema: [{
+            properties: {
+                method: {
+                    pattern: '^([hH][eE][aA][dD])|([gG][eE][tT])$',
```

Comment:
For better readability maybe use: `pattern: '^(get|GET|head|HEAD)$'`?

Reply:
It could also be gEt , etc.

Reply:
If this is an **ajv** schema, you may be able to use `regexp` and `flags` To handle case insensitivity:
https://github.com/epoberezkin/ajv-keywords#regexp

Reply:
> For better readability maybe use: pattern: '^(get|GET|head|HEAD)$'?

As @molant said, there is to much possibilities hehehe.

>If this is an ajv schema, you may be able to use regexp and flags To handle case insensitivity:

I didn't know that!! Thanks @pdehaan

Reply:
Ahhh, but it is another package... Due to the simplicity of the regex we need, I rather use the standard `pattern` instead of add a new package.

@webhintio/contributors thoughts?

Reply:
I think we can keep it as it is, but good to know something this package exists in case we need it in the future. Thanks @pdehaan for the pointer!